nrf9120-hal/flash_tfm.sh

```diff
@@ -0,0 +1,2 @@
+nrfjprog --family NRF91 --recover
+nrfjprog --family NRF91 --chiperase --verify --program tfm.hex
```

Comment:
Where does `tfm.hex` come from?

Reply:
Trusted Firmware M.  It replaces (includes) the Secure Partition Manager in newer versions of the Nordic SDK.  It is required to run in non-secure mode on the nrf91s.  It can be built with the SDK (e.g. the hello world TFM example).  It is not needed for nrf9120 support, and I didn't mean to include it in the PR.  I will remove it.  I was trying to do too many things at once yesterday when I submitted this.  Apologies for any confusion.